docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -238,6 +238,31 @@ will return a JSON object of the form:
 
 where 123456, 342332, etc are the candidates that exist for this project.
 
+#### 2.1.5 Single project electrophysiology recordings
+```
+GET /projects/$ProjectName/recordings
+```
+
+will return a JSON object of the form:
+```js
+{
+  "Recordings": [
+    {
+      "Candidate":"300167",
+      "PSCID":"OTT167",
+      "Visit":"V1",
+      "Visit_date":"2016-08-15",
```

Comment:
Is this different than the session Visit date?

Reply:
no, this field is derived from the Visit_date field of the session table.